demo/dynamic_embedding/movielens-100k-sync-estimator/README.md

```diff
@@ -0,0 +1,16 @@
+# A distributed training demo for `tfra.dynamic_embedding`:
```

Comment:
Changing name to movielens-100k-sync-estimator-with-horovod would be better.

Reply:
done.